src/Tizen.NUI/src/internal/Common/ProcessorController.cs

```diff
@@ -40,8 +40,12 @@ private ProcessorController() : this(true)
         {
         }
 
-        private ProcessorController(bool initializeOnConstructor) : this(initializeOnConstructor ? Interop.ProcessorController.New() : Interop.ProcessorController.NewWithoutInitialize(), true)
+        private ProcessorController(bool initializeOnConstructor) : this(Interop.ProcessorController.NewWithoutInitialize(), true)
```

Comment:
It seems to me that it always initializes Dali regardless of the bool argument.
I don't see any problem.